core/cap-0023.md

```diff
@@ -239,146 +293,248 @@ default:
+is clear that a `ClaimableBalanceEntry` cannot be a sub-entry of any of its
+potential claimants, because any account can be specified as a potential
+claimant and it is a security risk for accounts to be able to add sub-entries to
+other accounts. But why should the `ClaimableBalanceEntry` be an independent
+entity on the ledger rather than a sub-entry of the account that created it?
+There are three main benefits of this design:
 
 1. Sending accounts are not limited in the number of payments they can initiate
 2. Sending accounts can be merged even if they initiated payments that have not
-been claimed
-3. Receiving accounts always receive the reserve from the claimed `BalanceEntry`
-
-The third benefit facilitates a desirable optimization in which a `BalanceEntry`
-can be directly converted into a `TrustLine`. This is useful in some asset
-issuance workflows where the issuer wants to pay for the trustline but does not
-want the recipient to spend the funds prior to creating the trustline. Of course
-if the base reserve were raised after the `BalanceEntry` was created then the
-reserve received would not be sufficient to pay for the trustline, but they would
-at least still receive the reserve. But if the `BalanceEntry` were a sub-entry
-of the sending account and the base reserve were raised such that the sending
-account no longer satisfies the reserve requirement then the receiving account
-would receive no reserve (because accounts that do not satisfy the reserve
-requirement cannot have their native balance decrease). While this is an unlikely
-edge case, it does simplify the behavior of `BalanceEntry` by guaranteeing that
-the receiving account always receives both the reserve and the amount of asset.
-
-### BalanceEntry createdBy and claimableBy are accountIDs
-Each `BalanceEntry` can only be claimed by `claimableBy` or `createdBy` (exactly
-one of them depending on the ledger close time). As a consequence, it is possible
-for a `BalanceEntry` to become stranded. If `claimableBy` is merged and the
-private key is not known, then the `BalanceEntry` will never be claimable by the
-intended recipient. Similarly if `createdBy` is merged and the private key is not
-known, then the `BalanceEntry` will never be reclaimable by the sender.
-
-Suppose that we try to relax this requirement in order to avoid this downside. We
-could instead attach two public keys to the `BalanceEntry`, where the sender alone
-knows one of the private keys and the recipient alone knows the other. Which key is
-currently usable would be determined by the ledger close time. The operation to
-claim the `BalanceEntry` could then specify an account A and a signature over A
-from the key that is currently usable. This would allow the appropriate party to
-claim the `BalanceEntry` into any account that they control. But this would also
+   been claimed
+3. Receiving accounts always receive the reserve from the claimed
```

Comment:
This makes using claims vs normal sends many times more expensive (I think 100,000x more expensive). The additional cost might be prohibitive?